spec/biker_spec.rb

```diff
@@ -0,0 +1,70 @@
+require './lib/ride'
+require './lib/biker'
+
+include Ride
+
+class Biker
+  attr_reader :name,
+              :acceptable_terrain,
+              :max_distance
+
+  def initialize(name, acceptable_terrain)
+    @name = name
+    @acceptable_terrain = acceptable_terrain
+    @max_distance = max_distance 
+  end
+
+  it 'has a name' do
+    biker = Biker1.new("Kenny")
+    name = biker1.name
+    expect(name).to eq ("Kenny")
+  end  
+
+  it 'has an acceptable_terrain' do
+    acceptable_terrain = Acceptable_terrain.new(["gravel", "hills"])
```

Comment:
Acceptable_terrain is not a class, but rather an attribute for the Biker object. Be sure to create a new biker here, and then you can call `biker.acceptable_terrain` and make assertions about what it returns


Also, be careful about the interaction pattern. Our interaction pattern shows that acceptable_terrain is not something that needs to be passed in when we do `.new`. Instead, it starts as an empty array and it can be added to by using the `learn_terrain!` method